manifests/Snort/Snort/2.9.16.yaml

```
@@ -0,0 +1,17 @@
+Id: Snort.Snort
+Publisher: Snort
+Name: Snort
+Author: Snort
```

```suggestion
Id: Cisco.Snort
Publisher: Cisco Systems
Name: Snort
Author: Cisco Systems
```

Cisco purchased Sourcefire, Snort's original dev, in 2013. They currently develop and maintain this application.
Don't forget to change the folder name as well.